threads/README.md

```diff
@@ -1,2 +1,3 @@
-Threading Examples
+Threading Examples:
+Create an example program that shows how to create multiple threads. Put the code in the threads/ directory.
 =======
```

Comment:
this end sup looking funny (https://github.com/pradeep-k/adv-net-samples/blob/master/threads/README.md) because  the equal signs turn it into a headline.  Leave the headline of the document "Threading Examples" but then below add a short description of your thread_demo.c file.  In the future there will probably be other examples put in the folder, so this will let someone quickly find an example with the types of functions they are looking for.